web/pgadmin/browser/server_groups/servers/databases/schemas/collations/__init__.py

```diff
@@ -713,7 +704,6 @@ def sql(self, gid, sid, did, scid, coid, **kwargs):
         """
         json_resp = kwargs.get('json_resp', True)
         target_schema = kwargs.get('target_schema', None)
-
```

Comment:
Not sure why you've removed this blank lines everywhere but those were added for readability reasons. Check your patch once again.